entries/m/mexc.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "MEXC": {
+    "domain": "mexc.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.mexc.com/learn/article/17827791509545",
+    "categories": [
+      "cryptocurrencies"
+    ]
```

Comment:
```suggestion
    ],
    "regions": [
      "-ca",
      "-cu",
      "-kp",
      "-hk",
      "-ir",
      "-cn",
      "-sg",
      "-sd",
      "-sy",
      "-gb",
      "-us",
    ]
```

I was able to get this from [the terms](https://www.mexc.com/terms) before being geoblocked.

> You must ensure that you are not resident of Prohibited Countries. Currently, MEXC does not provide Services, nor do we accept registration of Users or trade applications, in the following countries: North Korea, Cuba, Sudan, Syria, Iran, Mainland China, Singapore, the United States, the United Kingdom, Hong Kong, Russian-controlled regions of Ukraine (currently including the Crimea, Donetsk, and Luhansk regions), Sevastopol, and Canada (altogether referred to as 'Prohibited Countries'). For the sake of clarification, the Prohibited Countries list is non-exclusive and is subject to change, at any time and from time to time, by MEXC in its absolute sole discretion may decide, taking into account legal and compliance considerations. You understand and acknowledge that if it is determined that you have given false representations of your location or place of residence, we reserve the right to take any appropriate actions in compliance with applicable laws and regulations, including termination of any Account immediately and liquidating any open positions. You undertake to inform MEXC at the earliest possible opportunity if you have become resident in any of the foregoing Prohibited Countries.